examples/storage_example/src/main.sw

```diff
@@ -1,21 +1,19 @@
 contract;
 
-use std::storage::{get, store};
+use std::{constants::ZERO_B256, storage::{get, store}};
 
 abi StorageExample {
     #[storage(write)]fn store_something(amount: u64);
     #[storage(read)]fn get_something() -> u64;
 }
 
-const STORAGE_KEY: b256 = 0x0000000000000000000000000000000000000000000000000000000000000000;
-
 impl StorageExample for Contract {
     #[storage(write)]fn store_something(amount: u64) {
-        store(STORAGE_KEY, amount);
+        store(ZERO_B256, amount);
```

Comment:
This shouldn't be used here, since keys are means to be 0, 1, 2, 3, etc. There's nothing special about the first one. `ZERO_B256` can be used e.g. for comparisons.

Reply:
IIRC, storage keys don't increment yet, but use a hash onion approach.
(I had to revert the PR which implemented `Add` for `b256`.)
Still, I think the suggestion here is a good one.